On The Map/Controller/AddLocationCompletionViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AddLocationCompletionViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/23/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import CoreLocation
+
+class AddLocationCompletionViewController: UIViewController, MKMapViewDelegate {
+   
+    var coordinate: CLLocationCoordinate2D = CLLocationCoordinate2D()
+    var link:String = ""
+    var mapString: String = ""
+
+    @IBOutlet weak var mapView: MKMapView!
+    
+    @IBAction func addLocation(_ sender: Any) {
+        Client.addStudentLocation(mapString: mapString, mediaURL: link, latitude: coordinate.latitude, longitude: coordinate.longitude) { (success: Bool, error:String?) in
+            guard success, error == nil else {
+                self.errorAlertMessage(title: "", message: error!)
```

Comment:
error alert message is missing a title, not a big deal but you could make a string extension to provide a default error message for these kind of cases

Reply:
good catch, not sure how I missed that.